pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "smaht-submitr"
-version = "1.6.3"
+version = "1.6.4"
```

Comment:
Maybe a minor version change is appropriate? (I assume there is no breaking change)